modules/core/src/system.cpp

```diff
@@ -508,6 +508,11 @@ int64 getCPUTickCount(void)
 
 #endif
 
+std::ostream& operator << (std::ostream& out, const TickMeter& tm)
```

Comment:
What kind of errors do you have? Did you try "static inline" without CV_EXPORTS?